website/docs/components/code-editor/partials/code/component-api.md

```diff
@@ -0,0 +1,90 @@
+## Component API
+
+This component uses [CodeMirror 6](https://codemirror.net/) under the hood.
```

Comment:
[suggestion] This is also included in the "How to use" section, does it need to also be here?

Reply:
Maybe delete it from the "How to use" section & leave it here since it relates more to the API?